gsoc/2022/mentors.md

```diff
@@ -20,6 +20,7 @@ layout: plain
 * Mario Lassnig [mario.lassnig@cern.ch](mailto:mario.lassnig@cern.ch) CERN
 * Javier Lopez-Gomez [Javier Lopez-Gomez](mailto:j.lopez@cern.ch) CERN
 * Axel Naumann [Axel Naumann](mailto:Axel.Naumann@cern.ch) CERN
+* Enrico Guiraud [Enrico Guiraud](mailto:enrico.guiraud@cern.ch) CERN
```

Comment:
could you put yourself into the alphabetical order?

Reply:
done